File: crystal-ball/src/main/java/org/strangeforest/tcb/model/TieBreakOutcome.java
package org.strangeforest.tcb.model;

public class TieBreakOutcome extends DiffOutcome {

	public TieBreakOutcome(double pServe, double pReturn) {
		super(7, 2, pointNo -> (pointNo / 2) % 2 == 0 ? pServe : pReturn);
	}
}

File: crystal-ball/src/main/kotlin/org/strangeforest/tcb/model/TieBreakOutcome.kt
package org.strangeforest.tcb.model

class TieBreakOutcome(
	pServe: Double,
	pReturn: Double
)
	: DiffOutcome(7, 2, { pointNo -> if (pointNo / 2 % 2 == 0) pServe else pReturn }
)
